reana_server/status.py

```diff
@@ -255,8 +267,10 @@ def get_status(self):
         return {
             "running": self.get_workflows_by_status(RunStatus.running),
             "finished": self.get_workflows_by_status(RunStatus.finished),
-            "stuck": self.stuck_workflows(),
+            "stuck in running": self.unschedulable_workflows(),
+            "stuck in pending": self.stuck_workflows(),
             "queued": self.get_workflows_by_status(RunStatus.queued),
+            "pending": self.get_workflows_by_status(RunStatus.pending),
```

Comment:
Perhaps we don't need this? It could give a false impression if just in the morning when the status report is calculated a few workflows are in pending, even if few seconds after are scheduled.